studio/components/interfaces/App/PortalToast.tsx

```diff
@@ -28,35 +28,42 @@ const PortalToast = () => (
     >
       {(t) => (
         <ToastBar toast={t} style={t.style}>
-          {({ icon, message }) => (
-            <>
-              {icon}
-              <div className="flex items-center">
-                <div
-                  className={`toast-message w-full ${
-                    t.type === 'loading' ? 'max-w-[350px]' : 'max-w-[260px]'
-                  }`}
-                >
-                  {message}
-                </div>
-                {t.type !== 'loading' && (
-                  <div className="ml-4">
-                    <Button
-                      className="!p-1"
-                      type="text"
-                      onClick={(event) => {
-                        event.preventDefault()
-                        event.stopPropagation()
-                        toast.dismiss(t.id)
-                      }}
-                    >
-                      <IconX size={14} strokeWidth={2} />
-                    </Button>
+          {({ icon, message }) => {
+            const isConsentToast = t.id === 'consent-toast'
+            return (
+              <>
```

Comment:
`possible fragment not needed`

Reply:
fragment required here cause of `{icon}` on the next line